server/server.js

```diff
@@ -1,6 +1,9 @@
 require("dotenv").config();
+// express: fast, minimailist framework for node.js
```

Comment:
nit: I like that you're researching the different libraries, and making notes on them. I wouldn't leave these comments here though - people can easily google `express` if they don't know what it is.

I'd say that a list of libraries / frameworks could be in a `.md` file within the `/server` folder, describing your library/framework & design decisions

Reply:
good idea